B5 Completion

m-emre opened the pull request:
Please refer to the attached files.

supermario commented:
rejected:B5
reason:account address not submitted, cannot verify on-chain baker update transactions

m-emre commented:
@supermario I have copied my account and transaction details below. Could you please re-verify them?

Account:
4RQVoXPui4aae9cA4pFVoPQNs667CCgJgG6nzP11HoF4BC8Y73
----------------------
Add baker
Transaction Hash:
5080fade985e34d049fb62212da4d551dc36f3418d81a28e8499e2a451e80068
Block Hash:
41527cb93dcd23cb5d6277b6b21c8c04e51c00bc066787c174d8ad5f0e683b1a
Added baker 936

----------------
Reward for baker 936
1c268bb0781830decf989b17c85c41f64ca7e3ab3d313873a7080da7005605d7
Award 1.597076 GTU to baker 936 at
4RQVoXPui4aae9cA4pFVoPQNs667CCgJgG6nzP11HoF4BC8Y73

---------------------------
Reward for baker 936
Block Hash:
41c4fbdf56a684c80986733c468a8c45c8a615e96dfdcb257eeeea6ecd115035
Details
Award 1.828178 GTU to baker 936 at
4RQVoXPui4aae9cA4pFVoPQNs667CCgJgG6nzP11HoF4BC8Y73

-------------------------
Add baker
Transaction Hash:
bd77acf3ef53540cb332d95e731769e84d47cc57d3f08643ab73f08bd9598361
Block Hash
929e9412c677e0bcc4f8163e1eb7ddccaa98c87f4b6ff4f6f3852c38da4bed49
Details
Added baker 1061

------------------------
Reward for baker 1061
Block Hash:
6babf8deab91daf6b8ef28001529bb4fb5f64be052f09b5a403ce72b819eaf6f
Details:
Award 1.459410 GTU to baker 1061 at
4pRsNPKiCKrpCFg4u7w3Nuvc9UtlhCUVixEifTFuN7qpVjn3FX

m-emre commented:
@supermario @bissembert1618 Could you please re-check my submissions plaese?

m-emre commented:
> Please refer to the attached files.

@supermario @bissembert1618 Could you please re-check my submissions plaese?

cryptovoyagr commented:
@supermario Can you please have a look at this ( B5 ) when you get the time? I know you are probably flooded with notifications, notifying you incase you missed it.

supermario commented:
rejected:B5
address:4RQVoXPui4aae9cA4pFVoPQNs667CCgJgG6nzP11HoF4BC8Y73
reason: no 'update baker account' or 'update baker key' transactions found on-chain

Hey @m-emre – I did check this again, but unfortunately the B5 requirements don't seem to be met.

You did pass on B4 though which I updated in that separate PR: https://github.com/Concordium/Testnet3-Challenges/pull/1195
